README.md

```diff
@@ -3,29 +3,48 @@
 [![NPM Package](https://img.shields.io/npm/v/@openzeppelin/test-helpers.svg)](https://www.npmjs.org/package/@openzeppelin/test-helpers)
 [![Build Status](https://travis-ci.com/OpenZeppelin/openzeppelin-test-helpers.svg?branch=master)](https://travis-ci.com/OpenZeppelin/openzeppelin-test-helpers)
 
-**JavaScript testing helpers for Ethereum smart contract development.** With support for Truffle and plain web3.js workflows.
+*Assertion library for Ethereum smart contract testing.* Make sure your contracts behave as expected!
 
-## Installation
+ * Check that [transactions revert](docs/modules/ROOT/pages/api.adoc#expect-revert) for the correct reason
+ * Verify [events](docs/modules/ROOT/pages/api.adoc#expect-event) where emitted with the right values
```

Comment:
I think we should say "verify _that_ events", to be consistent with the previous item.